botfront/docs/guide/deployment/installation.mdx

```diff
@@ -143,6 +143,19 @@ The following table shows the environment variables required by each service. Be
 | `API_KEY`            | GraphQL API key. You can then set the `authorization` header to the `API_KEY` value to perform GraphQL operations                                                                                        | `botfront`                |
 | `MODELS_LOCAL_PATH`  | Where the trained model returned by Rasa is stored locally. Defaults to `/app/models/` and should not be changed in a containerized environment. The Botfront Dockerfile exposes a volume with that path | `botfront` (optional)     |
 
+The following table list environment variables used to configure logging within botfront and botfront-api. Make those available as arguments or in the manifest files of your deployment
+
+| Environment variable            | Description                                                                                                                                                                | Allowed values                     | default              |
+|---------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------|----------------------|
+| `APPLICATION_LOG_TRANSPORT`     | Where the application log are written                                                                                                                                      | console, stackdriver               | console              |
+| `APPLICATION_LOG_LEVEL`         | From witch level the log are logged                                                                                                                                        | critical, error, warn, info, debug | info                 |
+| `APPLICATION_LOGGER_NAME`       | The name of the application logger on stackdriver                                                                                                                          | Any string                         | botfront_log_app     |
+| `MAX_LOGGED_ARG_LENGHT`         | Maximum lenght of a stringified callingArgs entry, for readability/data usage concerns. Used only when logging at info level and when APPLICATION_LOG_LEVEL is set to info | Any positive integer               | 1000                 |
```

Comment:
There is a typo here: MAX_LOGGED_ARG_LENGTH

Reply:
Thanks